avocado/core/__init__.py

```diff
@@ -178,6 +178,15 @@ def register_core_options():
         help_msg=help_msg,
     )
 
+    help_msg = "When this is enabled, avocado will disable all environment variables during exec testing"
```

Comment:
The commit message does not matches the changes in this version.  Also maybe simply stating:

Whether to run the exec-test with an empty environment (no variables set)